### ISSUE TITLE
Table naming for PostgreSql dialect

### ISSUE THREAD

**Issue by kohansey** [MoonStorm/FastCrud#135]
I am attempting to use FastCRUD to access records from third-party postgresql server.  Because its third-party, I have no control over the naming of tables or columns.  When I set the OrmConfiguration.DefaultDialect to PostgreSql I get an exception stating that the table can not be found.  If I take the SQL statement in the exception and run a query on the server, I get the same error message, however, if I remove the quotes around the table name, the query on the server executes fine.  How do I configure FastCRUD to not double quote table names when talking to the postgresql server?

**Comment by Gameghostify** [comment]
Also have this issue; there is no `SchemaAttribute` so I have to include it in my `TableAttribute`, like this:

```cs
[Table("user.accounts")]
public class UserModel
{
	[Key]
	[Column("id")]
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
	public int Id { get; set; }

	[Column("username")]
	public string Username { get; set; }
}
```

However, since it gets surrounded by quotes, the db throws an error that the relation can't be found (which makes sense, since `user.accounts` isn't a table - `accounts` is the only thing that should be quoted here)

**Comment by Gameghostify** [comment]
Alright

I've looked into the repository and found out that the OrmConventions class actually gets the schema name from the `TableAttribute`

If you had the same problem as I did and tried to access a table within a schema using something like `[Table("my_schema.my_table")]`, all you've got to do is use the `Schema` property of the `TableAttribute` like so:

`[Table("my_table", Schema = "my_schema")]`

---

However, if you weren't trying to access anything in a schema, I can't really explain where the error is coming from. It'd then greatly help if you could post the query that's being executed on the server in here (as well as the FastCrud methods you're calling)

**Comment by MoonStorm** [comment]
Yup, the `Schema` property inside the `Table` attribute should be used. There is another open case that might prevent the schema from being used properly. I'll take a look.

**Comment by MoonStorm** [comment]
Should be fixed in 3.3. Give it another go please.
